lib/normalizer.js

```diff
@@ -16,8 +16,8 @@ const projectPreprocessor = require("./projectPreprocessor");
  * @param {string} options.translator Translator to use
  * @returns {Promise} Promise resolving to tree object
  */
-async function generateProjectTree(options) {
-	const tree = await generateDependencyTree(options);
+async function generateProjectTree(options = {}) {
+	const tree = await this.generateDependencyTree(options);
```

Comment:
This is done to be able to mock the function, right?
I don't think that it's good practice though as they are just local functions and at the end, we assign both to the module.exports object. In case `generateProjectTree ` is called as local function, this would break.

Reply:
Good point, I missed that.

Reply:
> This is done to be able to mock the function, right?
> I don't think that it's good practice though as they are just local functions and at the end, we assign both to the module.exports object. In case `generateProjectTree ` is called as local function, this would break.

I have added a fix 87e9873 for this by wrapping the functions into a module. We will lose the facade in this case but will be able to test the module. Please check if the fix is sufficient

Reply:
Yep, that looks much better now 👍